src/app/static/src/js/components/publishReports/publishReports.vue

```diff
@@ -32,7 +41,19 @@
         props: ["reportsWithDrafts"],
         data() {
             return {
-                publishedOnly: false
+                publishedOnly: false,
+                expandClicked: 0,
```

Comment:
This was the neatest way I could think of transmitting event data from the parent component down to the child - just increment a prop value when the button is clicked, and watch for changes in the child. What do you think?

Reply:
Right, because it's not setting a property that should have a permanent effect it's just making a change that can be overridden by the child components. This seems like a good approach to me. I suppose an alternative would be to use a timestamp which might seem a little less arbitrary than a counter whose value doesn't really get used. But times are always a pain so it's probably much of a muchness.